<launch>
    <param name="source_frame" value="front_axle" />
    <param name="target_frame" value="map" />
    <param name="odom_frame" value="map"/>
    <param name="child_frame" value="front_axle"/>
    <node name="odom_pub" pkg="perception" type="odom_pub" output="screen">
    </node>

    <node name="gazebo_tf" pkg="perception" type="gazebo_tf" output="screen">
    </node>

</launch>
